lib/main.rb

```diff
@@ -0,0 +1,17 @@
+def bubble_sort(numbers)
+  sorted = true
+
+  (numbers.length - 1).times do |i|
```

Comment:
This is just a personal preference. Leave it or take it :P
```suggestion
  (0...numbers.length).each do |i|
```

Reply:
take a look!!